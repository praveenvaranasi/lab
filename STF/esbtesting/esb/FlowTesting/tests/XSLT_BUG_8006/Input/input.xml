<ns1:BookStore xmlns:ns1="http://www.books.org">
   <ns1:Book>
      <ns1:Title>Title</ns1:Title>
      <ns1:Author>Author</ns1:Author>
      <ns1:Date>Date</ns1:Date>
      <ns1:ISBN>ISBN</ns1:ISBN>
      <ns1:Publisher>Publisher</ns1:Publisher>
   </ns1:Book>
</ns1:BookStore>
