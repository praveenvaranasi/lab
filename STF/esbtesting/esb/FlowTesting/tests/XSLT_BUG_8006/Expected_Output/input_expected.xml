<?xml version="1.0" encoding="UTF-8"?>
<ns1:BookStore xmlns:ns1="http://www.books.org" xmlns:ns2="http://www.w3.org/2001/XMLSchema">
  <ns1:Book>
    <ns1:Title>Titl</ns1:Title>
    <ns1:Author>Author</ns1:Author>
    <ns1:Date>Date</ns1:Date>
    <ns1:ISBN>ISBN</ns1:ISBN>
    <ns1:Publisher>Publisher</ns1:Publisher>
  </ns1:Book>
</ns1:BookStore>
